rocketpy/rocket/aero_surface.py

```diff
@@ -1889,3 +1892,180 @@ def all_info(self):
         """
         self.prints.all()
         return None
+
+
+class Airbrakes(AeroSurface):
```

Comment:
I'm getting an annoying spelling warning everytime it is written "airbrake" (without separation). 

Apparently, both "airbrake" and "air brake" is accpeted, but the second option is more common. Indeed, searching on wikipedia, it is way more common to find it with separation (https://en.wikipedia.org/wiki/Air_brake_%28aeronautics%29).

Also, I wonder if it should be in plural or not. 
- Fins and RailButtons are plural
- Tail and NoseCone are not
  
Any suggestions here?

Therefore, I suggest changing the class name to either `AirBrake` or `AirBrakes`

Reply:
Also, I would be happy if we change every instance of "airbrakes" to "air_brakes" :)

Reply:
Good catch! I will change it to AirBrakes, since this seems to be the most common way to refer to this type of system

Reply:
> Also, I would be happy if we change every instance of "airbrakes" to "air_brakes" :)

Interesting note: during initial development, this was discussed quite extensively, and we even changes from AirBrakes to Airbrakes.

We got this from some EuRoC technical reports... But I have to agree that your references seem more popular!

Reply:
Question here, should we change `add_airbrakes` to `add_air_brakes`?